plugins/system/sortbyfield/sortbyfield.php

```diff
@@ -101,6 +101,10 @@ public function onContentPrepareForm(Form $form, $data): bool
 
 	public function onContentBeforeSave(?string $context, $table, $isNew = false, $data = null): bool
 	{
+		// Joomla 4 Media Manager freaks out when the plugin is enabled; skip this plugin if the context is com_media.
+		if(str_contains($context,'com_media')) {
```

Comment:
`str_contains` is PHP 8.0 and later. It also matches all contexts with com_media in them while you want to only match those starting with it. Change that to 
```php
if (substr($context, 0, 10) === 'com_media.') {
```